templates/vertical-grid/page.html.hbs

```diff
@@ -33,18 +33,18 @@
         {{!-- {{> templates/vertical-grid/collapsible-filters/markup/filterlink }} --}}
         {{!-- {{> templates/vertical-grid/collapsible-filters/markup/viewresultsbutton }} --}}
       </div>
-      <!-- Uncomment the following div if you want to include filters, facets, or sort options  -->
-      {{!-- <div class="Answers-filtersWrapper js-answersFiltersWrapper CollapsibleFilters-inactive"> --}}
-        {{!-- {{> templates/vertical-grid/markup/sortoptions }} --}}
-        {{!-- {{> templates/vertical-grid/markup/filterbox }} --}}
-        {{!-- {{> templates/vertical-grid/markup/facets }} --}}
-      {{!-- </div> --}}
       <div class="Answers-results js-answersResults">
         {{> templates/vertical-grid/markup/spellcheck }}
         {{> templates/vertical-grid/markup/verticalresults }}
         {{> templates/vertical-grid/markup/pagination }}
         {{!-- {{> templates/vertical-grid/markup/qasubmission }} --}}
       </div>
+      <!-- Uncomment the following div if you want to include filters, facets, or sort options  -->
```

Comment:
Is there some way we could avoid switching the order of these divs and also the flex-direction: row-reverse? I was thinking it might be possible to add a container div for Answers-results and Answers-filtersWrapper, that had display: flex and justify-content: center. The row-reverse + flex-wrap is a little counter intuitive to me (though definitely cool!) and we might be able to get away with something a little simpler

Reply:
Would the we be able to avoid swapping the order of the divs by setting the flex order?

Reply:
I just had the same thought! Testing the change right now